src/main/java/teammates/common/datatransfer/FeedbackSessionAttributes.java

```diff
@@ -484,4 +484,27 @@ public Date getSessionEndTime() {
     public String getSessionName() {
         return this.feedbackSessionName;
     }
+    
+    /*
+     * Comparator to sort by endTime in descending order
+     */
+    public static final Comparator<FeedbackSessionAttributes> compareByEndTimeDesc = new Comparator<FeedbackSessionAttributes>() {
```

Comment:
already exists one comparator that did the job
https://github.com/TEAMMATES/repo/blob/master/src/main/java/teammates/common/datatransfer/SessionAttributes.java#L58